packages/react/src/components/AudioPlayer/AudioPlayerVolumeControl.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import Button from '../../internal/vendor/carbon-components-react/components/Button/Button';
+
+import { DDS_AUDIO_PLAYER } from '../../internal/FeatureFlags';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import root from 'window-or-global';
+
+import settings from 'carbon-components/es/globals/js/settings';
+
+import Slider from '../../internal/vendor/carbon-components-react/components/Slider/Slider';
+
+import VolumeDown32 from '@carbon/icons-react/es/volume--down/32';
+import VolumeMute32 from '@carbon/icons-react/es/volume--mute/32';
+import VolumeUp32 from '@carbon/icons-react/es/volume--up/32';
+
+const { prefix } = settings;
+
+const AudioPlayerVolumeControl = ({
+  kalturaDigitalPlayer,
+  audioVolume,
+  setAudioVolume,
+  displayVolumeControl,
+  setDisplayVolumeControl,
+}) => {
+  const handleAudioVolumeChange = volume => {
+    volume = root.Number.parseFloat(volume.toString().substring(0, 3));
+    if (kalturaDigitalPlayer) {
+      kalturaDigitalPlayer.sendNotification('changeVolume', volume);
+      setAudioVolume(volume);
+    }
+  };
+
+  const renderVolumeButtonIcon = () => {
+    if (audioVolume === 0) {
+      return VolumeMute32;
+    } else if (audioVolume > 0 && audioVolume < 0.8) {
+      return VolumeDown32;
+    }
+    return VolumeUp32;
+  };
+
+  const isTooltipVisible = () => {
+    if (displayVolumeControl) {
+      return `${prefix}--force-tooltip-hidden ${prefix}--menu--open`;
+    }
+    return '';
+  };
```

Comment:
Whenever possible we prefer to use `cx` to set classes, e.g.
```suggestion
  const buttonClasses = cx({
    [`${prefix}--force-tooltip-hidden ${prefix}--menu--open`]: displayVolumeControl,
  });
```